grudge/discretization.py

```diff
@@ -71,30 +75,41 @@ def __init__(self, array_context, mesh, order=None,
                         "'quad_tag_to_group_factory' must be given")
 
             quad_tag_to_group_factory = {
-                    QTAG_NONE: PolynomialWarpAndBlendGroupFactory(order=order)}
+                    DISCR_TAG_BASE: PolynomialWarpAndBlendGroupFactory(order=order)}
         else:
             if order is not None:
                 quad_tag_to_group_factory = quad_tag_to_group_factory.copy()
-                if QTAG_NONE in quad_tag_to_group_factory:
+                if DISCR_TAG_BASE in quad_tag_to_group_factory:
                     raise ValueError("if 'order' is given, "
                             "'quad_tag_to_group_factory' must not have a "
-                            "key of QTAG_NONE")
+                            "key of DISCR_TAG_BASE")
 
-                quad_tag_to_group_factory[QTAG_NONE] = \
+                quad_tag_to_group_factory[DISCR_TAG_BASE] = \
                         PolynomialWarpAndBlendGroupFactory(order=order)
 
+        # FIXME: Get rid of QTAG_NONE in downstream grudge applications
+        # Is this too aggressive/invasive?
+        if QTAG_NONE in quad_tag_to_group_factory.keys():
```

Comment:
```suggestion
        if QTAG_NONE in quad_tag_to_group_factory:
```

Reply:
Done in https://github.com/inducer/grudge/pull/83/commits/826f66fc3b3b6e7cc67dc16e40946bbc23361e9e